src/formatter.ts

```diff
@@ -98,13 +100,22 @@ export async function formatter(results: ESLint.LintResult[]): Promise<string> {
     results = filterResults(results);
   }
 
-  return await printResults(results, {
+  let resultsToPrint = await printResults(results, {
     formatTodoAs,
     updateTodo,
     includeTodo,
     shouldCleanTodos,
     todoInfo,
   });
+
+  if (compactAfterProcess) {
+    const { compacted } = compactTodoStorageFile(baseDir);
+
+    resultsToPrint &&= resultsToPrint + "\n"
+    resultsToPrint += `Removed ${compacted - todoInfo.cleanedUp} todos from .lint-todo storage file`;
+  }
```

Comment:
I'm wondering if we should always display the message "x todo were removed" as long as the FLAG/CONST is given (even when none were been removed) or only if flag is given + some where actually removed